docs/src/components/DocumentMetadata/index.tsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { type ReactNode } from 'react';
+import styles from './styles.module.css';
+
+interface Props {
+  lastUpdatedAt: string;
+  authorInfo: string;
+  lastTranslatedAt?: string;
+};
+
+export default function DocumentMetadata({
+  lastUpdatedAt,
+  authorInfo,
+  lastTranslatedAt
+}: Props): JSX.Element {
+  return (
+    <div className={styles.block}>
+      <i className={styles.info}>Last updated at: <b>{lastUpdatedAt}</b></i>
```

Comment:
What about we make those:

- Links to the actual commit (visualized on Github)
- Relative times (e.g. _3 days ago_)

Reply:
New looking style
<img width="547" alt="image" src="https://user-images.githubusercontent.com/14299604/184385418-6ba438b2-b506-4ebe-a2ae-6a9514cac1e1.png">
wdyt?

Reply:
What about not showing the date at all, and making the '2 minutes ago' the link 😅? We should have done that while pairing I guess, this back and forth feels unproductive. Requirements unclear. Sorry.

Reply:
Other than that,  I think it's amazing :D